src/libsystemd/sd-bus/bus-socket.c

```diff
@@ -728,6 +729,7 @@ int bus_socket_exec(sd_bus *b) {
                 (void) reset_signal_mask();
 
                 close_all_fds(s+1, 1);
+                make_null_stdio();
```

Comment:
hmm, can you elaborate on the page rproblem with this? I think it would be good if any errors by ssh would actually be visible in the output, hence leaving stderr around sounds like a good idea?

normally the bus connection should be closed down first, which would cause ssh to exit, and hence make th pager exit. why does this not happen in this case?